docs/Bots.md

```diff
@@ -420,6 +420,72 @@ FIXME
 
 * * *
 
+### Squelcher
+
+The configuration defines a TTL based on matching fields. The bot looks in the
+event table if an event has been sent to the customer in the time frame defined
+by the TTL using the IP, classification type and classification identifier.
+If an event has been found, we set notify to false, otherwise to true.
+Only events with notify flag set to true are considered for the lookups.
+
+Edge cases:
+If no ASN is present, we ignore the event and set the flag to false.
+If no IP is present, but an FQDN , we always set the flag to true.
+if `extra._origin` equals `"dnsmalware"`, the flag is always set to false.
+
+#### Information:
+* `name:` squelcher
+* `lookup:` postgres
+* `public:` yes
+* `cache (redis db):` -
+* `description:` Sets the `notify` field to true or false depending on past notifications
+
+#### Configuration Parameters:
+
+ * `autocommit`: use transactions per statement (`true`) or per connection (`false`)
+ * `configuration_path`: path to the squelcher configuration file, e.g. `"/opt/intelmq/etc/squelcher.conf"`
+ * `database`: postgres database
+ * `host"` postgres host
+ * `password`: postgres password
+ * `port`: postgres port (usually 5432)
+ * `sslmode`: e.g. `"require"`
+ * `table`: postgres table
+ * `user`: postgres user
+
```

Comment:
Can we have a parameter "time_interval" or so? This shall specify how many days it will look back into time.